app/src/main/java/com/kickstarter/viewmodels/MessageHolderViewModel.kt

```diff
@@ -48,7 +48,7 @@ interface MessageHolderViewModel {
         ActivityViewModel<MessageViewHolder?>(environment),
         Inputs,
         Outputs {
-        private val currentUser: CurrentUserType?
+        private val currentUser: CurrentUserTypeV2?
```

Comment:
very minor, but currentUser cant be null (due to requireNotNull(*)) so it doesn't need to be nullable here

Reply:
I'll fix in a subsequent PR! Thanks!